### PR TITLE
Fix build badge and fix readme formatting

### PR DESCRIPTION
**Pull request by cewert**
## Changes
- Fixes build badge - see [here ](https://github.com/badges/shields/issues/8671) for more information
- Fixes build badge link - now takes you to the `build.yml` workflow results as intended
- Fix markdown formatting (linting errors from vscode extension)
- Remove outdated warning about needing a specific brighterscript version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,42 +2,45 @@
...
 <h3 align="center">
-Simple, mocha-and junit inspired, flexible, fun Brightscript test framework for ROKU apps
+Simple, mocha and junit inspired, flexible, and fun Brightscript test framework for Roku apps
 </h3>
 <p align="center">
   <img src="images/exampleImage.png" alt="Mocha test framework" />
 </p>
 
-[![build](https://img.shields.io/github/workflow/status/georgejecook/rooibos/build.svg?logo=github)](https://github.com/georgejecook/rooibos/actions?query=workflow%3Abuild)
+[![build](https://img.shields.io/github/actions/workflow/status/georgejecook/rooibos/build.yml?branch=master)](https://github.com/georgejecook/rooibos/actions/workflows/build.yml)
```
Comment (TwitchBronBron) [inline README.md]: can you add the `logo=github` back into the querystring as well?


```suggestion
[![build](https://img.shields.io/github/actions/workflow/status/georgejecook/rooibos/build.yml?logo=github&branch=master)](https://github.com/georgejecook/rooibos/actions/workflows/build.yml)
```

File: README.md [hunk 2]
```diff
@@ -2,42 +2,45 @@
...
-Rooibos is an independent open-source project, maintained exclusively by volunteers.
+Rooibos is an independent open-source project maintained exclusively by volunteers.
 
-You might want to help! Get in touch via the slack group, or raise issues.
+You might want to help! Get in touch via the [slack group](https://join.slack.com/t/rokudevelopers/shared_invite/zt-4vw7rg6v-NH46oY7hTktpRIBM_zGvwA) or [raise issues](https://github.com/georgejecook/rooibos/issues/new).
 
 ## Sample project
-https://github.com/georgejecook/rooibos-roku-sample
+
+<https://github.com/georgejecook/rooibos-roku-sample>
```
Comment (TwitchBronBron) [inline README.md]: What's the point of these `<` `>` chars? I don't think it's of much benefit.

Comment (cewert) [reply README.md]: Because of this linting rule - [MD034](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md034.md)

TL;DR - `Rationale: Without angle brackets, the URL isn't converted into a link by many Markdown parsers.`

I can change it if you'd like.

Comment (TwitchBronBron) [reply README.md]: Yeah, I think this one isn't quite as intuitive, and most of us are editing and consuming these exclusively within a github/vscode context so the `<` and `>` are just kind of in the way. :) So let's remove them for this PR.

**Review by TwitchBronBron — APPROVED** [review]
